### PR TITLE
Sprinkle some sleeps to try and improve e2e flakiness

### PR DESCRIPTION
**Pull request by smukherj1**

### CODE REVIEW

**Comment by smukherj1** [comment]
@fejta Passed in one shot! Maybe I got lucky?

File: examples/hellogrpc/e2e-test.sh [hunk 1]
```diff
@@ -55,9 +55,14 @@ check_msg() {
...
   while [[ -z $(get_lb_ip) ]]; do
     echo "service has not yet received an IP address, sleeping for 5s..."
-    sleep 5
+    sleep 10
   done
 
+  # Wait some more for after IP address allocation for the service to come
+  # alive
+  echo "Got IP Adress! Sleeping 30s more for service to come alive..."
+  sleep 30
```
Comment (chrislovecnm) [inline examples/hellogrpc/e2e-test.sh]: We have had to loop multiple times waiting on LBs.  I would recommend putting in a couple of 30 second loops.

**Comment by chrislovecnm** [comment]
We need a rebase

**Comment by chrislovecnm** [comment]
@fejta is prow not doing the needs rebase label on this repo?

**Comment by smukherj1** [comment]
> @fejta is prow not doing the needs rebase label on this repo?

I see an option to "Update branch" which is likely do the rebase. I'll check this in for now and address your comment about looping the sleep if the context cancellation errors show up again

**Comment by smukherj1** [comment]
Ah it retriggered the tests. Let's see what happens shall we

**Comment by chrislovecnm** [comment]
@smukherj1 prow has an option to check if PRs need a rebase and tags the PR automatically.

**Comment by smukherj1** [comment]
Oh snap

**Pull request merged**